### DISCUSSION TITLE
how to add 2nd camers with ha.io? Not using docker.

### DISCUSSION THREAD

**Discussion by goulaobangzi** [blakeblackshear/frigate#792] [Ask A Question] [answered]
Can't find a way to add another camera? 

The first one works great.

**Comment by Minglarn** [comment, marked as answer]
Mind if you share your config.yml (frigate.yml)?
Just add a extra camera under camera:
Se below...

'cameras:
#############################
# ENTREN                    #
#############################
  frigate_entren:
    ffmpeg:
      inputs:
        - path: >-
            rtmp://192.168.1.58/bcs/channel0_main.bcs?channel=0...
          roles:
            - clips
            - rtmp
      

#############################
# PARKERING                 #
#############################
  frigate_parkering:
    ffmpeg:
      inputs:
        - path: 'rtsp://....@192.168.1.72:88/videoMain'
          roles:
            - clips
            - rtmp
     ...'

**Reply by goulaobangzi** [reply]
I think my object mask was the issue. It always gives me an error msg.

**Reply by goulaobangzi** [reply]
Can you set up an object mask for the individual camera? 

Thanks~

**Reply by goulaobangzi** [reply]
You know what. It's my index issue. I am so bad. 
Now problem resolved.
